content/en/platform/corda/5.2/deploying-operating/deployment/upgrading521/_index.md

```diff
@@ -0,0 +1,517 @@
+-u uniqueness=UNIQUENESS_WORKER -u flowMapper=FLOW_MAPPER_USER -u persistence=PERSISTENCE_USER \
+-u verification=VERIFICATION_WORKER preview
+```
+
+{{< note >}}
+You do not need a connection to Kafka to perform this step. The YAML file represents the correct state for Kafka topics based on the platform version the tool was built against. It does not include information about the current state of any running Kafka instance. Any differences should be identified by the Kafka instance administrator or the automated tool that processes the YAML file.
+{{< /note >}}
+
+### Corda CLI Managed Kafka Updates
+
+For clusters using the development prerequisites, you can obtain Kafka login credentials and the SSL certificate from the cluster secret. This is the same to what you did in section [Migrate Data From Kafka Topics to the Cluster Database](#migrate-data-from-kafka-topics-to-the-cluster-database), so if the file was already generated in that step, it can be reused here, allowing you to skip to running the plugin. If the cluster was deployed differently, it is assumed the upgrader already has this information. You can pass a standard Kafka properties file for authentication to the tool using the `-k` parameter. You can generate such a file as follows:
+
+```
+KAFKA_PASSWORD=$(kubectl get secret -n corda prereqs-kafka -o go-template='{{ index .data "admin-password" | base64decode }}')
+kubectl get secret -n corda prereqs-kafka -o go-template='{{ index .data "ca.crt" | base64decode }}' > ./kafka_config/ca.crt
+cat > ./kafka_config/props.txt <<_EOF
+security.protocol=SASL_SSL
+sasl.mechanism=PLAIN
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username=\"admin\" password=\"${KAFKA_PASSWORD}\";
+ssl.truststore.location=/kafka_config/ca.crt
+ssl.truststore.type=PEM
+_EOF
+```
+
+To have the tool connect to the cluster and perform the necessary additions to ACLs, use the following code:
+
+```
+docker run -v $(pwd)/kafka_config:/kafka_config --add-host prereqs-kafka:host-gateway $CLI_DOCKER_IMAGE \
+topic -b=prereqs-kafka:9092 -k=/kafka_config/props.txt create connect
+```
+
+Refer the [Appendix A](#appendix-a-development-prerequisites-for-connecting-to-kafka) for information on how R3 uses `--add-host` to enable the command running in the Docker container to access Kafka for development prerequisites. The Appendices also provide alternatives for users using Corda CLI installed locally rather than Docker container.
+
+## Launch the Corda 5.2.1 Workers
+
+If Corda 5.2 was deployed using Corda Helm chart, you can deploy Corda 5.2.1 the same way. This updates the deployments with the new image versions and scales them to the defined replica counts. Corda version is overridden at the command line and selecting the 5.2.1 Corda Helm chart defaults to the 5.2.1 worker images. If you provide your own values in a YAML file, ensure it does not refer to 5.2 images.
+
```

Comment:
We should link to the original helm chart instructions here I think, rather than give an example. Also I see these instructions refer to 5.2.0 and we need some to refer to 5.2.1.

https://docs.r3.com/en/platform/corda/5.2/deploying-operating/deployment/deploying.html#download-the-corda-helm-chart

Reply:
Linked to the helm chart instructions. They should be updated today/tmorrow once I get content from @wzur-r3